html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+  <h3>Pokemon: Squirtle</h3>
+  <p> © Pokemon. ® Nintendo </p>
```

Comment:
Please use the HTML entities for these characters rather than pasting them into the document.